R/shared-methods.R

```diff
@@ -320,8 +324,13 @@ DataClass <- R6::R6Class(
         region_codes = self$supported_region_codes
       )
 
+      if (is.null(self$data$filtered)) {
```

Comment:
rather than do this why not just dup clean into filtered its just a pointer if no changes.